src/js/edu-benefits/containers/ReviewPage.jsx

```diff
@@ -0,0 +1,191 @@
+      onFieldsInitialized={pageProps.onFieldsInitialized}
+      onUpdateVerifiedStatus={pageProps.onUpdateVerifiedStatus}
+      pageLabel={label}
+      updatePath={path}
+      component={<Component
+          data={pageProps.data}
+          onStateChange={pageProps.onStateChange}
+          initializeFields={pageProps.onFieldsInitialized}/>}
+      reviewComponent={<ReviewComponent data={pageProps.data}/>}/>);
+}
+
+class ReviewPage extends React.Component {
+  render() {
+    let content;
+
+    if (this.props.uiData.isApplicationSubmitted) {
+      content = (
+        // TODO(crew): We need to figure out why the css isn't working here.
+        <div className="usa-alert usa-alert-success">
+          <div className="usa-alert-body">
+            <h3 className="usa-alert-heading">You have submitted your application for education benefits!</h3>
+            <p className="usa-alert-text">We are processing your application. You should receive a phone call from the VA in the next week.</p>
+            <p className="usa-alert-text">If you do not receive a call from the VA within a week, or you have questions, call 1-877-222-VETS (8387) and press 2.</p>
+          </div>
+        </div>
+      );
+    } else {
+      content = (<div>
+        <p>Please make sure all your information is correct before submitting your application.</p>
+        <ReviewPanel
+            pageProps={this.props}
+            label="Personal Information"
+            path="/veteran-information/personal-information"
+            component={PersonalInformationFields}
+            reviewComponent={PersonalInformationReview}/>
+        <ReviewPanel
+            pageProps={this.props}
+            label="Contact Information"
+            path="/veteran-information/contact-information"
+            component={ContactInformationFields}
+            reviewComponent={ContactInformationReview}/>
+        <ReviewPanel
+            pageProps={this.props}
+            label="Secondary Contact"
+            path="/veteran-information/secondary-contact"
+            component={SecondaryContactFields}
+            reviewComponent={SecondaryContactReview}/>
+        <ReviewPanel
+            pageProps={this.props}
+            label="Direct Deposit"
+            path="/veteran-information/direct-deposit"
+            component={DirectDepositFields}
+            reviewComponent={DirectDepositReview}/>
+        <ReviewPanel
+            pageProps={this.props}
+            label="Benefits Selection"
+            path="/benefits-eligibility/benefits-selection"
+            component={BenefitsSelectionFields}
+            reviewComponent={BenefitsSelectionReview}/>
+        <ReviewPanel
+            pageProps={this.props}
+            label="Military Service"
+            path="/military-history/military-service"
+            component={MilitaryServiceFields}
+            reviewComponent={MilitaryServiceReview}/>
+        {this.props.data.seniorRotcCommissioned.value === 'Y'
```

Comment:
It might be a good to pull this logic from `routes.jsx` for DRYness purposes (though that does make it less readable)